samples/sqs/simple/Sqs_5/Sender/Program.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Threading.Tasks;
+using NServiceBus;
+
+class Program
+{
+    static async Task Main()
+    {
+        Console.Title = "Samples.Sqs.SimpleSender";
+        #region ConfigureEndpoint
+
+        var endpointConfiguration = new EndpointConfiguration("Samples.Sqs.SimpleSender");
+        var transport = endpointConfiguration.UseTransport<SqsTransport>();
+        transport.S3("bucketname", "my/key/prefix");
+
+        #endregion
+
+        transport.Routing().RouteToEndpoint(typeof(MyCommand), "Samples.Sqs.SimpleReceiver");
+        endpointConfiguration.EnableInstallers();
+
+        var endpointInstance = await Endpoint.Start(endpointConfiguration);
+
+        await SendMessages(endpointInstance);
+
+        await endpointInstance.Stop();
+    }
+
+    static async Task SendMessages(IMessageSession messageSession)
+    {
+        Console.WriteLine(@"Press
```

Comment:
Nitpicking: maybe `c`, `C` and `e`, `E` to be consistent with other samples?

Reply:
I was thinking about that, but it felt that whatever chars we select, it's scattered all over the board (not sure about the uppercase idea tbh), so I went with 1-4 so that the options are easy to locate. No strong feelings on this though 👍